web/src/components/speaker/Links.vue

```diff
@@ -1,22 +1,46 @@
 <template>
   <div class="mt-3">
-    <social-button :link="twitter">
-      <i class="fab fa-twitter" />
-    </social-button>
-    <social-button :link="linkedIn">
-      <i class="fab fa-linkedin-in" />
-    </social-button>
-    <social-button :link="facebook">
-      <i class="fab fa-facebook-f" />
-    </social-button>
-    <social-button :link="website">
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on }">
+        <social-button :link="twitter">
+          <i class="fab fa-twitter" v-on="on" />
+        </social-button>
+      </template>
       <!--eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
-      <v-icon>web</v-icon>
-    </social-button>
-    <social-button :link="priorPresentation">
-      <!--eslint-disable-next-line @intlify/vue-i18n/no-raw-text -->
-      <v-icon>ondemand_video</v-icon>
-    </social-button>
+      <span>Twitter</span>
+    </v-tooltip>
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on }">
+        <social-button :link="linkedIn">
+          <i class="fab fa-linkedin-in" v-on="on" />
+        </social-button>
+      </template>
+      <span>LinkedIn</span>
+    </v-tooltip>
+    <v-tooltip bottom>
+      <template v-slot:activator="{ on }">
+        <social-button :link="facebook">
+          <i class="fab fa-facebook-f" v-on="on" />
+        </social-button>
+      </template>
+      <span>Facebook</span>
```

Comment:
Ahh, I notice we should be using internationalization on these, but let's fix in a later PR.

Reply:
https://github.com/WWCodeTokyo/speak-her-db/issues/228